### PR TITLE
Add iFrame Resizer to LTI Tools

### PR DESCRIPTION
**Pull request by orbruno**
This patch adds the [iFrame Resizer](https://github.com/davidjbradshaw/iframe-resizer) to the LTI tools. This allows integrations to dynamically adjust the IFrame size to the size of the included content, making it look more like a native tool.

### CODE REVIEW

File: modules/lti/public/index.html [hunk 1]
```diff
@@ -30,9 +30,10 @@
 
       You can add webfonts, meta tags, or analytics to this file.
       The build step will place the bundled scripts into the <body> tag.
-
+
```
Comment (KatrinIhler) [inline modules/lti/public/index.html]: Unintentional whitespace change!

File: modules/lti/public/iframeResizer.contentWindow.min.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+//# sourceMappingURL=iframeResizer.contentWindow.map
```
Comment (KatrinIhler) [inline modules/lti/public/iframeResizer.contentWindow.min.js:10]: Whats up with this line?

Comment (lkiesow) [reply modules/lti/public/iframeResizer.contentWindow.min.js:10]: That's how a minified JavaScript library often looks like… ?

File: modules/lti/public/index.html (hunk 1 above)
Comment (lkiesow) [reply modules/lti/public/index.html]:
```suggestion

```

File: modules/lti/public/iframeResizer.contentWindow.min.js (hunk 2 above)
Comment (KatrinIhler) [reply modules/lti/public/iframeResizer.contentWindow.min.js:10]: Okay, I was confused why it was commented out, but then nevermind. :D

**Comment by lkiesow** [comment]
This fixes  #1892

**Review by LukasKalbertodt — CHANGES_REQUESTED** [review]
I tried this locally and it works... kind of. 

The series tool is a bit broken because the `<header>` (e.g. "Results 1-2 of 2") has a margin that pushes the `<body>` down without increasing its height. That leads to the bottom of the page being hidden (half of the pagination buttons are hidden). And since this library also sets `overflow: hidden`, users cannot even scroll anymore. That's unfortunate.

</img>


This could be fixed by either fixing our CSS or using [this](https://github.com/davidjbradshaw/iframe-resizer/blob/master/docs/iframed_page/options.md#heightcalculationmethod--widthcalculationmethod) (which curiously also says one should first try fixing the CSS :D). This should certainly be fixed before merging this.

The iframe also doesn't seem to resize when clicking on a video, which is unexpected.

---

Finally, this is an NPM project which means we have a proper dependency management. We shouldn't include minified JS files in our repository and include them directly in the HTML. Instead, `npm install iframe-resizer --save` adds the dependency and adding `import 'iframe-resizer';` in `src/index.ts` includes it into the project. (It's worth noting that the iframe-resizer project description really doesn't explain this part very well...).

I tried this locally and this works, so that's certainly the preferred way of including the library.

**Comment by LukasKalbertodt** [comment]
Superseded by #2562. This PR has some problems still and we don't expect the author to address those, so I just created a new PR.

**Pull request closed without merging**